lib/html.rb

```diff
@@ -16,39 +16,127 @@
 # you may find current contact information at www.suse.com
 
 class Html
-  def self.escape_javascript(object)
-    object.gsub(/([\\'"])/, "\\\\\\1")
+  module Helpers
+    def scope_help(scope)
+      text = File.read(File.join(Machinery::ROOT, "plugins", "#{scope}/#{scope}.md"))
+      Kramdown::Document.new(text).to_html
+    end
+
+    def diff_to_object(diff)
+      diff = Machinery.scrub(diff)
+      lines = diff.lines[2..-1]
+      diff_object = {
+        file: diff[/--- a(.*)/, 1],
+        additions: lines.select { |l| l.start_with?("+") }.length,
+        deletions: lines.select { |l| l.start_with?("-") }.length
+      }
+
+      original_line_number = 0
+      new_line_number = 0
+      diff_object[:lines] = lines.map do |line|
+        line = ERB::Util.html_escape(line.chomp).
+          gsub("\\", "&#92;").
+          gsub("\t", "&nbsp;"*8)
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Surrounding space missing for operator '*'.